plugins/quay-common/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@janus-idp/backstage-plugin-quay-common",
+  "version": "0.1.0",
```

to save a step with MSR failing on us, please make this 1.0.0 instead of 0.1 
```suggestion
  "version": "1.0.0",
```

We'll need to MANUALLY release this plugin at version 1.0.0 to allow subsequent releases to work with this monorepo.

Alternatively, since this is a NEW PLUGIN, it should be contributed to the https://github.com/backstage/community-plugins repo.

---

Updated the version to 1.0.0.